libs/ui-lib/lib/ocm/components/clusterConfiguration/review/ReviewPreflightChecks.tsx

```diff
@@ -103,26 +104,26 @@ const PreflightCheckInfo = ({
 const getCheckIcon = (validationStatuses: string[]) => {
   if (validationStatuses.includes('failure') || validationStatuses.includes('error')) {
     return (
-      <Icon size="sm" status="danger">
-        <ExclamationCircleIcon />
+      <Icon status="danger">
```

Comment:
I don't know if it's the same case for you, but unless I specify `size="md"`, all of the icons except for the Success one, show up strangely small:

![image](https://github.com/openshift-assisted/assisted-installer-ui/assets/87187179/c7858d8b-9cd7-4927-a1bc-84fc2008dedc)

Reply:
@jgyselov I'll take a look

Reply:
I see that if we delete size="sm" from icons, they look well:
![image](https://github.com/openshift-assisted/assisted-installer-ui/assets/11390125/3ed6e792-bffc-4716-bcbe-47e0a072cbbe)

Reply:
That's weird but ok :shrug:

Reply:
I took a look to uhc-portal and they use this format:
`  const iconProps = {
    className: 'clusterstate',
    size: 'md',
  };
  `
  And then:
  
   <Icon {...iconProps}>
        <ExclamationCircleIcon color={dangerColor.value} />
      </Icon>